applications/Makefile

```diff
@@ -22,7 +22,11 @@ all: \
 	gocr.install \
 	gps_ekf.install \
 	license_plate_detection.install \
-	resize_image.install
+	resize_image.install \
+	scratch_storage_get.install \
```

Comment:
A little strange to have these apps in `wasm_apps` as they are quite sledge specific, but I'm sure that applies to multiple apps there. No problem here, just a little cognitive dissonance.